modules/integration/tests-ui-integration/src/test/java/org/wso2/am/integration/ui/tests/VisibilityTestCase.java

```diff
@@ -0,0 +1,227 @@
+package org.wso2.am.integration.ui.tests;
+
+/**
+ * Created by nisala on 3/20/15.
+ */
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.wso2.am.integration.ui.tests.util.APIMTestConstants;
+import org.wso2.carbon.automation.extensions.selenium.BrowserManager;
+import org.wso2.am.integration.ui.pages.home.HomePage;
+import org.wso2.am.integration.ui.pages.login.LoginPage;
+import org.wso2.am.integration.ui.pages.tenant.TenantHomePage;
+import org.wso2.am.integration.ui.pages.tenant.TenantListpage;
+
+import java.util.List;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+
+
+/**
+ * Created by nisala on 2/23/15.
+ */
+public class VisibilityTestCase extends AMIntegrationUiTestBase {
```

Comment:
This parent class has been renamed to APIMIntegrationBaseTest. Please update your PR accordingly.